### DISCUSSION TITLE
More regions for Vercel storage

### DISCUSSION THREAD

**Discussion by Thinkscape** [vercel/community#2264] [Idea]
So Storage has [just been announced](https://vercel.com/blog/vercel-storage) which uses Neon under the hood for its postgres variant.

Unfortunately Neon is limited to just handful of regions (most in continental USA)
Because of data sovereignty and performance we'd need a region where our business, and our customers, are located.

Is there a roadmap for introducing more AWS regions?

**Comment by amyegan** [comment]
Hi @Thinkscape! Is there a specific region, or range of regions, that would work better for you?

**Reply by Thinkscape** [reply]
Most of our team and our customers are in `ap-southeast-2` Sydney region.

**Reply by amyegan** [reply]
Thanks for letting me know! I can't make any guarantees about future availability, but I let the team know you're interested in having `ap-southeast-2` as an option so the they know which locations have higher demand when planning for new regions 🙂

**Comment by qt-pixels** [comment]
Hello, I've just been notified Neon DB have now opened up `ap-southeast-2`. I'm writing here to see when Vercel will enable this configuration settings? 

FYI @amyegan , @Thinkscape

**Reply by jinbe** [reply]
It seems that the setting is there now. I just created a database in syd1 but it's been over an hour and the instance hasn't started up.

**Comment by Thinkscape** [comment]
Just created instance in `syd` via Vercel "storage" UI.
Working, can connect, run queries.

Thanks!
